scripts/machine-launch-script.sh

```diff
@@ -1,83 +1,264 @@
+    # MADV_NOHUGEPAGE in a call to madvise.  See:
+    # https://man7.org/linux/man-pages/man2/madvise.2.html
+    # https://conda-forge.org/docs/maintainer/knowledge_base.html#using-centos-7
+    # obvi this would need to be made linux-specific if we supported other MacOS or Windows
+    CONDA_PACKAGE_SPECS+=( "kernel-headers_linux-64>=2.6.38" )
+    # firemarshal deps
+    CONDA_PACKAGE_SPECS+=( rsync )
+    # cross-compile glibc 2.28+ deps
+    CONDA_PACKAGE_SPECS+=( make )
+    # other misc deps
+    # conda-forge splits out the shared library into libzlib
+    # not sure if we need both, but I'm including them for now
+    CONDA_PACKAGE_SPECS+=(
+        sbt \
+        ca-certificates \
+        mosh \
+        gmp \
+        mpfr \
+        mpc \
+        zlib \
+	libzlib \
+        vim \
+        git  \
+        openjdk \
+        gengetopt \
+        libffi \
+        expat \
+        libusb1 \
+        ncurses \
+        cmake \
+        graphviz \
+        expect
+        dtc \
+	verilator==4.034 \
+    )
+
+    # python packages
+    # While it is possible to install using pip after creating the
+    # conda environment, pip's dependency resolution can conflict with
+    # conda and create broken environments.  It's best to use the conda
+    # packages so that the environment is consistent
+    CONDA_PACKAGE_SPECS+=( \
+	boto3==1.20.21 \
+        colorama==0.4.3 \
+	argcomplete==1.12.3 \
+	python-graphviz==0.19 \
+	pyparsing==3.0.6 \
+	numpy==1.19.5 \
+	kiwisolver==1.3.1 \
+	matplotlib==3.3.4 \
+	pandas==1.1.5 \
+	awscli==1.22.21 \
+	pytest==6.2.5 \
+	pytest-dependency==0.5.1 \
+	pytest-mock==3.7.0 \
+	moto==2.2.19 \
+	pyyaml==5.4.1 \
+	mypy==0.931 \
+	types-pyyaml==6.0.4 \
+	boto3-stubs==1.21.6 \
+	botocore-stubs==1.24.7 \
+	mypy-boto3-s3==1.21.0 \
+    )
+
+    if [[ "$CONDA_ENV_NAME" == "base" ]]; then
+	# we default to installing into the base environment against best practice because it
+	# simplifies adding our environment as the default to the system-wide /etc/profile.d/conda.sh
+	CONDA_SUBCOMMAND=install
+	CONDA_ENV_BIN="${CONDA_INSTALL_PREFIX}/bin"
+    else
+	CONDA_SUBCOMMAND=create
+	CONDA_ENV_BIN="${CONDA_INSTALL_PREFIX}/envs/${CONDA_ENV_NAME}/bin"
+    fi
+
+    # to enable use of sudo and avoid modifying 'secure_path' in /etc/sudoers, we specify the full path to pip
+    CONDA_PIP_EXE="${CONDA_ENV_BIN}/pip"
+
+    # to enable use of sudo and avoid modifying 'secure_path' in /etc/sudoers, we specify the full path to conda
+    $SUDO "${CONDA_EXE}" "$CONDA_SUBCOMMAND" $DRY_RUN_OPTION -n "$CONDA_ENV_NAME" -y "${CONDA_PACKAGE_SPECS[@]}"
+
+
+    # Install python packages using pip that are not available from conda
+    #
+    # Installing things with pip is possible.  However, to get
+    # the most complete solution to all dependencies, you should
+    # prefer creating the environment with a single invocation of
+    # conda
+    PIP_PKGS=( \
+	fab-classic==1.19.1 \
+	mypy-boto3-ec2==1.21.9 \
```

Comment:
```suggestion
	mypy-boto3-ec2==1.21.9 \
	sure==2.0.0 \
```
Noticed that I somehow missed `sure` when I grabbed the other new packages from where-to-build.